src/posix/platform/openthread-posix-config.h

```diff
@@ -58,33 +58,37 @@
 #endif
 
 /**
- * @def OPENTHREAD_POSIX_CONFIG_RCP_UART_ENABLE
+ * @def OPENTHREAD_POSIX_CONFIG_DAEMON_ENABLE
  *
- * Define as 1 to enable UART interface to RCP.
+ * Define to 1 to enable POSIX daemon.
  *
  */
-#ifndef OPENTHREAD_POSIX_CONFIG_RCP_UART_ENABLE
-#define OPENTHREAD_POSIX_CONFIG_RCP_UART_ENABLE 0
+#ifndef OPENTHREAD_POSIX_CONFIG_DAEMON_ENABLE
+#define OPENTHREAD_POSIX_CONFIG_DAEMON_ENABLE 0
 #endif
 
 /**
- * @def OPENTHREAD_POSIX_CONFIG_RCP_SPI_ENABLE
+ * RCP bus UART.
  *
- * Define as 1 to enable SPI interface to RCP.
+ * @note This value is also for simulated UART bus.
  *
  */
-#ifndef OPENTHREAD_POSIX_CONFIG_RCP_SPI_ENABLE
-#define OPENTHREAD_POSIX_CONFIG_RCP_SPI_ENABLE 0
-#endif
+#define OT_POSIX_RCP_BUS_UART 1
 
 /**
- * @def OPENTHREAD_POSIX_CONFIG_DAEMON_ENABLE
+ * RCP bus SPI.
  *
- * Define to 1 to enable POSIX daemon.
+ */
+#define OT_POSIX_RCP_BUS_SPI 2
+
+/**
+ * @def OPENTHREAD_POSIX_RCP_BUS
+ *
+ * This setting configures what type of RCP bus to use.
  *
  */
-#ifndef OPENTHREAD_POSIX_CONFIG_DAEMON_ENABLE
-#define OPENTHREAD_POSIX_CONFIG_DAEMON_ENABLE 0
+#ifndef OPENTHREAD_POSIX_RCP_BUS
+#define OPENTHREAD_POSIX_RCP_BUS OT_POSIX_RCP_BUS_UART
```

Comment:
nit: any reason why `OPENTHREAD_POSIX_RCP_BUS` rather than `OPENTHREAD_POSIX_CONFIG_RCP_BUS`?

Do we want a config check to make sure this is set to either SPI or UART?

Reply:
Currently this is checked in radio.cpp, does it working for you?

Reply:
> Currently this is checked in radio.cpp, does it work for you?

Yes, I think that is fine. Thanks.